prosessering/src/main/java/no/nav/familie/prosessering/internal/ScheduledTaskService.kt

```diff
@@ -17,4 +19,11 @@ class ScheduledTaskService(private val taskRepository: TaskRepository) {
 
         tasks.forEach { taskRepository.save(it.klarTilPlukk(BRUKERNAVN_NÅR_SIKKERHETSKONTEKST_IKKE_FINNES)) }
     }
+
+    @Scheduled(cron = "0 0 9 1/1 * ?")
```

Comment:
flisespikking, men en kommentar for oss som ikke snakker flytende cron at dette er "At 09:00:00am, every day starting on the 1st, every month" elns.